sim/midas/src/main/cc/rtlsim/Makefrag-verilator

```diff
@@ -37,6 +38,11 @@ override VERILATOR_FLAGS := \
 	-Wno-DECLFILENAME \
 	-Wno-VARHIDDEN \
 	-Wno-UNDRIVEN \
+	-Wno-WIDTHTRUNC \
+	-Wno-UNUSEDPARAM \
+	-Wno-UNOPTFLAT \
+	-Wno-PINCONNECTEMPTY \
+	-Wno-fatal \
```

Comment:
Why is this needed? What Verilog was having these warnings... I think we should keep erroring on warning for now.